server/helpers/modelHelpers.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable require-jsdoc */
+export function automationStatus(DataTypes) {
+  return {
+    status: {
+      type: DataTypes.ENUM,
+      values: ['success', 'failure'],
+      allowNull: false,
+    },
+    statusMessage: {
+      type: DataTypes.STRING,
+      allowNull: true,
+      defaultValue: null,
+    },
+    automationId: DataTypes.STRING,
```

Comment:
I don't think `automationId` should be inside this function(`automationStatus`) that is just supposed to return(from the naming) the `automation status` section of the models.

Reply:
Thanks for the point out. Will rename the function to `baseAutomationFields`